Source/DiabloUI/ui_item.h

```diff
@@ -22,8 +22,9 @@ enum UiType : uint8_t {
 	UI_EDIT,
 };
 
-enum UiFlags : uint16_t {
+enum class UiFlags {
 	// clang-format off
+	NONE            = 0,
 	UIS_SMALL       = 1 << 0,

```

Comment:
Since all the line with usages are changed with this PR it would be nice if you could also adjust the naming of the values at the same time :)

```suggestion
	None        = 0,
	Small       = 1 << 0,
```
(`FontSmall` is also an option)

Reply:
Done. Used prefixes to show related flags, the Kerning flag is one I'm not happy with the name of but couldn't think of anything more suitable.

Reply:
You just made this PR very enticing :)